sunbeam-python/sunbeam/provider/local/commands.py

```diff
@@ -1225,6 +1287,16 @@ def join(
                     join_mode=True,
                     manifest=manifest,
                 ),
+                LocalConfigSRIOVStep(
+                    client, name, jhelper, deployment.openstack_machines_model, manifest
```

Comment:
accept_defaults is not passed as an argument

Reply:
good point, updating